scripts/js/queries.js

```diff
@@ -26,6 +26,17 @@ var filters = [
   "reply",
   "dnssec",
 ];
+var doDnssec = false;
```

Comment:
```suggestion
var doDNSSEC = false;
```

Only stylistic, but maybe we should keep DNSSEC all uppercase?